8.x/theme-development/galleries.md

```diff
@@ -1,3 +1,138 @@
 ## Template galleries
 
-[[toc]]
\ No newline at end of file
+[[toc]]
+
+In the Larammerce Template Engine, any section that is repetitive and the website admin wants to create a list of content and edit it uses the gallery.
```

Comment:
and the website admin wants to create a list of content and edit it uses the gallery.
->
and it's needed to be managed by the website admin to create/update/delete a list of them; Can be implemented by the template engine `galleries`.